src/entity/Entity.php

```diff
@@ -330,12 +330,20 @@ public function hasNoClientPredictions() : bool{
 	 *
 	 * Setting this flag will also disable player movement inputs, but this should not be relied on, as cheat clients
 	 * will be able to bypass it.
+	 * @deprecated USE setImmobile($value)
 	 */
 	public function setNoClientPredictions(bool $value = true) : void{
 		$this->noClientPredictions = $value;
 		$this->networkPropertiesDirty = true;
 	}
 
+
+	public function setImmobile(bool $value): void {
+		$this->noClientPredictions = $value;
+		$this->networkPropertiesDirty = true;
+	}
+
+
```

Comment:
yes it should be simple to use but well done nonetheless, otherwise I don't understand why recreate a function why just rename the function `setNoClientPredictions` ?

Reply:
Symply delete your created setImmobile and rename `setNoClientPredictions` to `setImmobile`

Reply:
because the but is not break other plugin

Reply:
no, but we can change the api

Reply:
I tend to agree, but why not solve the problem of making a truly immobile entity, such as not taking any more damage or not moving at all ?

Reply:
setImmobile is just being imobile otherwise could have added setImmortel